Expired/Views/SettingView.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  SettingView.swift
+//  Expired
+//
+//  Created by Karl Michael Reyes on 2023-02-06.
+//
+
+import SwiftUI
+
+struct SettingView: View {
+    @Environment(\.managedObjectContext) private var viewContext
+    @EnvironmentObject var productStore: ProductStore
+    @State private var showingDeleteAlert: Bool = false
+    @State private var showingSuccessAlert: Bool = false
+    var body: some View {
+        // Input here for Sandeep's part
+        List {
+            Section(header: Text("Archive List")) {
+                Text("test")
+            }
+
+            Section(header: Text("Data Management")) {
+                Group {
+                    Button("Remove all products") {
+                       showingDeleteAlert = true
+                    }
+                    .alert("Are you sure you want to remove all products? \n \n Deleting it will not be recoverable.", isPresented: $showingDeleteAlert) {
+                        Button("No", role: .cancel) {
+                            
+                        }
+                        Button("Yes", role: .destructive, action: {
+                            showingSuccessAlert = productStore.deleteAll(viewContext)
+                        })
+                    }
```

Comment:
```suggestion
                    .alert("Are you sure you want to delete all data?", isPresented: $showingDeleteAlert, actions: {
                        Button("No", role: .cancel) { }
                        Button("Yes", role: .destructive, action: {
                            deleteSuccess = productStore.deleteAll(viewContext)
                            showingDeleteResultAlert = true
                        })
                    }, message: {
                        Text("Deleting it will not be recoverable")
                    })
```

- Separate the message: "Deleting it will not be recoverable" from title with the `message` parameter, which gives a different UI
- `productStore.deleteAll(viewContext)` can return `true` or `false`, `deleteSuccess` is a more intuitive name than `showingSuccessAlert` for indicating the result of deletion
- `showingDeleteResultAlert` implies that the feedback alert will be displayed regardless of the result of deletion